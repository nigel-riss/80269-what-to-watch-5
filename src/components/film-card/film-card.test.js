import React from 'react';
import renderer from 'react-test-renderer';
import FilmCard from './film-card.jsx';
import filmsMock from '../../mocks/films.js';


const noop = () => {};

it(`Should render correctly`, () => {
  const tree = renderer
    .create(
        <FilmCard
          film={filmsMock[0]}
          isHovered={false}
          onMouseEnter={noop}
          onMouseLeave={noop}
          renderVideo={noop}
        />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
