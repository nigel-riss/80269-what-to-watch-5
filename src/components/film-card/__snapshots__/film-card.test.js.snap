// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should FilmCard render correctly 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onClick={[Function]}
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  style={
    Object {
      "backgroundColor": "#ffffff",
    }
  }
>
  <div
    className="small-movie-card__image"
  />
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
    >
      The Aviator
    </a>
  </h3>
</article>
`;
